Include repository protection rules (#132)
pyansys/dev-guide: doc/source/how-to/repository-protection.rst

Repository protection
=====================

Handling repositories also implies handling sensitive information, especially
when looking into workflows. Access to private servers to acquire Ansys product
licenses, secrets handling and so on. Thus, it is important that the PyAnsys
libraries have good protection rules implemented.

In the following sections, different safety measures are presented.

General configuration
---------------------

Being an owner/admin of a repository gives you access to the ``Settings`` menu.

> If one accesses:
> 
>     ``Repository`` >> ``Settings`` >> ``General``
> 
> That way, one accesses the *General configuration* of the repository. The PyAnsys
> team recommends the following measures in the **Pull Requests** (PRs) section:

Reviewer: ```suggestion
One accesses the *General configuration* of the repository by:

    ``Repository`` >> ``Settings`` >> ``General``

The PyAnsys team recommends the following measures in the **Pull Requests** (PRs) section:
```
